### PR TITLE
sweet: init at 1.10.5

### PR DESCRIPTION
**Pull request by LovingMelody**
###### Motivation for this change
Sweet theme is a popular theme on [r/unixporn](https://www.reddit.com/r/unixporn/), so I added created a package for it.

###### Things done

- [ ] Tested using sandboxing ([nix.useSandbox](http://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](http://nixos.org/nix/manual/#sec-conf-file) on non-NixOS linux)
- Built on platform(s)
   - [x] NixOS
   - [ ] macOS
   - [ ] other Linux distributions
- [] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [ ] Tested compilation of all pkgs that depend on this change using `nix-shell -p nix-review --run "nix-review wip"`
- [ ] Tested execution of all binary files (usually in `./result/bin/`)
- [ ] Determined the impact on package closure size (by running `nix path-info -S` before and after)
- [x] Ensured that relevant documentation is up to date
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

### CODE REVIEW

File: maintainers/maintainer-list.nix [hunk 1]
```diff
@@ -2490,6 +2490,12 @@
     githubId = 40521440;
     name = "Haruka Akiyama";
   };
+  fuzen = {
+    email = "me@fuzen.cafe";
+    github = "fuzen-py";
+    githubId = 17859309;
+    name = "Fuzen";
+  };
```
Comment (jonringer) [inline maintainers/maintainer-list.nix]: should be a separate commit

```
maintainers: add fuzen
```

File: pkgs/top-level/all-packages.nix [hunk 2]
```diff
@@ -17556,7 +17556,7 @@ in
   shaderc = callPackage ../development/compilers/shaderc { };
 
   shades-of-gray-theme = callPackage ../data/themes/shades-of-gray { };
-
+  sweet = callPackage ../data/themes/sweet { };
```
Comment (jonringer) [inline pkgs/top-level/all-packages.nix]: please add spaces
```suggestion

  sweet = callPackage ../data/themes/sweet { };
  
```

**Comment by jonringer** [comment]
just curious, how is this susposed to be tested?

**Comment by Fuzen-py** [comment]
I'm not sure how the testing processes for GTK themes works I am rather new to NixOS, I just installed the package and checked if the theme worked.

**Comment by jonringer** [comment]
@worldofpeace are you familiar with themes?

**Comment by worldofpeace** [comment]
> @worldofpeace are you familiar with themes?

Absolutely familiar :+1: .

You can test this by installing it into your nix-profile, themes are loaded from XDG_DATA_DIRS.
If someone prefers to not modify their system state they could do
```
GTK_THEME=$theme_name XDG_DATA_DIRS="$XDG_DATA_DIRS:$nix_store_path_to_theme/share" $some_gtk_program
```

or use the gtk debugger and select the theme from there.

**Comment by worldofpeace** [comment]
I've tested like
```
XDG_DATA_DIRS="$XDG_DATA_DIRS:$(nix-build -A sweet)/share" "$(nix-build -A gtk3.dev)/bin/gtk3-widget-factory"
```

and `Ctrl+Shift+d` -> Debugger Window -> Visual -> GTK Theme

File: pkgs/data/themes/sweet/default.nix [hunk 3]
```diff
@@ -0,0 +1,47 @@
...
+      sha256 = "0a7mh1pgvi8w1ahsmvgnmpdawm30lcjqk4zqvg0lqadsd04dn4h1";
+    })
+    (fetchurl {
+      url    = "https://github.com/EliverLara/Sweet/releases/download/v${version}/Sweet-mars.zip";
+      sha256 = "0n2dkl35qrik10wvhvkayyra987p03g56pxhz5kc73cbsl5zd96l";
+    })
+    (fetchurl {
+      url    = "https://github.com/EliverLara/Sweet/releases/download/v${version}/Sweet.zip";
+      sha256 = "0wwmc3wj2pjg4kimfkvcsimk3s4s7l7k000vxqi8yjlfs70f273c";
+    })
```
Comment (worldofpeace) [inline pkgs/data/themes/sweet/default.nix]: Please don't vertical align `=`. It's not a style used in nixpkgs (or rather, should).

File: pkgs/data/themes/sweet/default.nix [hunk 4]
```diff
@@ -0,0 +1,47 @@
...
+
+  nativeBuildInputs = [ unzip ];
+
+  sourceRoot = ".";
+
+  installPhase = ''
+    mkdir -p $out/share/themes/
+    cp -a Sweet* $out/share/themes/
+    rm $out/share/themes/*/{LICENSE,README*}
+  '';
```
Comment (worldofpeace) [inline pkgs/data/themes/sweet/default.nix]:
```suggestion
  '';
  
```

File: pkgs/data/themes/sweet/default.nix [hunk 5]
```diff
@@ -0,0 +1,47 @@
...
+  meta = with stdenv.lib; {
+    description = "Light and dark colorful Gtk3.20+ theme";
+    homepage    = "https://github.com/EliverLara/Sweet";
+    license     = licenses.gpl3;
+    maintainers = with maintainers; [ fuzen ];
+    platforms   = platforms.linux;
+  };
```
Comment (worldofpeace) [inline pkgs/data/themes/sweet/default.nix]: same as https://github.com/NixOS/nixpkgs/pull/74966/files#r356387267

File: pkgs/data/themes/sweet/default.nix [hunk 6]
```diff
@@ -0,0 +1,47 @@
...
+
```
Comment (worldofpeace) [inline pkgs/data/themes/sweet/default.nix]: trailing newline

**Review by worldofpeace — APPROVED** [review]

**Comment by worldofpeace** [comment]
Thanks @Fuzen-py :star2:
